wasm-page-template/script.js

```diff
@@ -0,0 +1,26 @@
+import init from "../out/thetawave.js";
```

Comment:
```suggestion
import init from "./thetawave.js";
```
We serve/deploy the "out/" directory as basically the page/site root.

https://github.com/thetawavegame/thetawave/blob/c5b82223919e557e1ceb3b34af4805321f40420b/.github/workflows/web_release.yml#L58

Locally I get "GET http://10.0.0.110:8080/out/thetawave.js net::ERR_ABORTED 404 (Not Found)" when running in Chrome.

Maybe we could move the wasm instructions to its own section, but the [CONTRIBUTING.md](https://github.com/thetawavegame/thetawave/blob/c5b82223919e557e1ceb3b34af4805321f40420b/CONTRIBUTING.md?plain=1#L31) gives one way to test this locally  



Run 
```bash
./build_wasm.sh
```
Then serve the `out/` directory with some HTTP server.

For example

```bash
python3 -m http.server -d out/
```

Reply:
Yikes, I forgot to delete that 😅 that was there only for testing, nice catch